src/components/ProductCard.js

```diff
@@ -186,6 +187,7 @@ const ProductCard = ({
           )}
           <Title gitHidden={!hasRepoData}>{name}</Title>
           <Description>{description}</Description>
+          {note.length > 0 && <Description>Note: {note}</Description>}
```

Comment:
I don't think having these notes directly on the cards makes sense. Seems far too in the weeds for the users we are targeting.

Reply:
Hmm, I think they still have a place on these cards. Does seem a bit dishonest leaving it out to me.